mw-webapp/src/stories/button/Button.stories.ts

```diff
@@ -0,0 +1,17 @@
+import {Button} from "./Button";
```

Comment:
is it possible to use absolute path here?

Reply:
Done